### PR TITLE
fix: 🐛 Switch to orienting the volume actor differently

### PR DESCRIPTION
**Pull request by JamesAPetts**
Switch to orienting the volume actor differently depending on its
acquisition direction, rather than filling up the volume differently.

### CODE REVIEW

**Comment by JamesAPetts** [comment]
@pieper , whenever you get a chance, please let me know if this is alright. I've tested in OHIF and it produces the same results as the previous PR. Thanks :)

File: src/lib/getImageData.js [hunk 1]
```diff
@@ -87,27 +87,10 @@ export default function getImageData(imageIds, displaySetInstanceUid) {
...
-    case 'coronal':
-      imageData.setDimensions(xVoxels, zVoxels, yVoxels);
-      imageData.setSpacing(xSpacing, zSpacing, ySpacing);
-      break;
-    case 'axial':
-      imageData.setDimensions(xVoxels, yVoxels, zVoxels);
-      imageData.setSpacing(xSpacing, ySpacing, zSpacing);
-      break;
-  }
+  imageData.setDirection(direction);
```
Comment (pieper) [inline src/lib/getImageData.js]: Ahhh, sooo much nicer!

File: src/VTKViewport/vtkInteractorStyleMPRSlice.js [hunk 2]
```diff
@@ -142,14 +140,12 @@ function vtkInteractorStyleMPRSlice(publicAPI, model) {
...
+      //let mapper = model.volumeMapper;
       // get the mapper if the model is actually the actor, not the mapper
-      if (!model.volumeMapper.getInputData && model.volumeMapper.getMapper) {
-        mapper = model.volumeMapper.getMapper();
-      }
-      let volumeCoordinateSpace = vec9toMat3(
-        mapper.getInputData().getDirection()
-      );
+      //if (!model.volumeMapper.getInputData && model.volumeMapper.getMapper) {
+      //  mapper = model.volumeMapper.getMapper();
```
Comment (pieper) [inline src/VTKViewport/vtkInteractorStyleMPRSlice.js]: Do you still need to leave this dead code for some reason?

Comment (JamesAPetts) [reply src/VTKViewport/vtkInteractorStyleMPRSlice.js]: Ooh good catch, I commented it out whilst refactoring but didn't delete it.

**Comment by mix3d** [comment]
@JamesAPetts I know this has already been merged, but the reason for the volumeCoordinate space was for volumes who's data was at oblique angles but you still wanted to set the Coronal to the volume's coronal orientation. Using the `volume.direction()`, if the values are strange, the same input view `orientation`s will look different depending. Most often an issue with MRs (in our sample data, at least).

I solved that by actually forcing the data to the cartesian coordinates direction, then using the direction data to modify the camera angle only for the RAS compass labels.

**Pull request merged**